### PR TITLE
Update Redis README

### PR DESCRIPTION
**Pull request by grant**
Related to #1032, b/133157483

### CODE REVIEW

File: memorystore/redis/README.md [hunk 1]
```diff
@@ -13,3 +13,21 @@ Follow the instructions in [this guide](https://cloud.google.com/memorystore/doc
...
+redis-server
+```
+
+In another terminal, run the Node application:
+
+```sh
+node server.js
+```
+
+Visit the following `localhost` URL to test your server that connects to Redis and observe a persistant "Vistor number".
```
Comment (ace-n) [inline memorystore/redis/README.md]: Typo: `vistor` -> `visitor` (add `i`)

File: memorystore/redis/README.md [hunk 2]
```diff
@@ -13,3 +13,21 @@ Follow the instructions in [this guide](https://cloud.google.com/memorystore/doc
 ## Running on Google App Engine Flex
 
 Follow the instructions in [this guide](https://cloud.google.com/memorystore/docs/redis/connect-redis-instance-flex) to deploy the sample application on GAE Flex.
+
+## Running Locally
+
+To run this sample locally, follow these instructions:
+
+In one terminal, [run Redis](https://redis.io/topics/quickstart):
```
Comment (ace-n) [inline memorystore/redis/README.md]: Is it better to refer to Redis Labs' [hosted redis solution](https://redislabs.com) here? (Maybe a question for the relevant PM.)

Comment (grant) [reply memorystore/redis/README.md]: I think the official Redis quickstart is better than that site for learning about what Redis is.

**Comment by grant** [comment]
@ace-n Please add more comments or approve :)

**Pull request merged**